app/models/movie_party.rb

```diff
@@ -4,8 +4,5 @@ class MovieParty < ApplicationRecord
   has_many :attendees, dependent: :destroy
   has_many :users, through: :attendees
 
-  # belongs_to :user
```

Comment:
Is there a reason to remove this line? I could be wrong, but I thought that it should belong to a user.